admin/apple-actions/index/class-get.php

```diff
@@ -52,16 +52,7 @@ public function perform() {
 		}
 
 		// Get the article from the API.
-		try {
```

Comment:
I removed this so that we keep the data removal intentional. Since GET requests are performed in more unrelated places than I anticipated.

Reply:
interesting here